contrib/pg_tde/Makefile

```diff
@@ -23,6 +23,8 @@ relocate \
 subtransaction \
 tablespace \
 vault_v2_test \
+cluster \
+partition_table \
```

Comment:
These should be added in alphabetical order with the caveat mentioned in the comment above, Plus you also need to add the tests to `meson.build`.

Reply:
Fixed